### ISSUE TITLE
Fix accessibility for screen reader users

### ISSUE THREAD

**Issue by TheQuinbox** [eramdam/BetterTweetDeck#765]
**Is your feature request related to a problem? Please describe.**
TweetDeck is mostly accessible to screen reader users, such as myself. However, there are still some issues. For example, the new tweet and search buttons are unlabeled, as well as the links towards the top of the page, and I have no idea what the random little number in the navigation bar is, either.
**Describe the solution you'd like**
Screen reader accessibility should be improved (e.g. add labels to buttons, links, etc). If you don't want this to clutter the visual interface, an aria-label attribute should work just fine.
**Describe alternatives you've considered**
Just keep it how it is, or try reporting it to the official TweetDeck team. I've tried the latter, and gotten basically nothing. And the former wouldn't be the end of the world, it's certainly doable, but just not the best experience.
**Additional context**

**Comment by TheQuinbox** [comment]
Also, for what it's worth, the new TweetDeck seems to have fixed these problems, but we have no idea when this will be rolling out to everyone.

**Comment by eramdam** [comment]
I consider Better TweetDeck to be in a feature freeze state due to the impending release of the newer TweetDeck and will instead shift my focus to building an alternative client. As such, I'm marking this issue as closed for the time being.
See the following issue for details #848
